extensions/cornerstone/src/getHangingProtocolModule.ts

```diff
@@ -184,6 +184,15 @@ const mprAnd3DVolumeViewport = {
           },
           required: true,
         },
+        {
+          attribute: 'Modality',
+          constraint: {
+            equals: {
```

Comment:
consider contains or containsI: ['ct', 'mr']

Reply:
the preset we use it not for MR, it is CT bone, so until we add ui for changing that I guess it is fine